bin/phase-helpers.sh

```diff
@@ -810,7 +810,7 @@ __eapi4_src_install() {
 
 __eapi6_src_prepare() {
 	if [[ $(declare -p PATCHES 2>/dev/null) == "declare -a"* ]]; then
-		[[ -n ${PATCHES[@]} ]] && eapply "${PATCHES[@]}"
+		[[ -n ${PATCHES[*]} ]] && eapply "${PATCHES[@]}"
```

Comment:
This is a meaningless change from my coding style to yours. No value for anyone, especially given that `[@]` is used on the same line.

Reply:
The reasoning is that `[@]` actually means "expand as separate words" while `[*]` means "expand as a single word." Consequently, [the semantics of this expansion may be misleading to some readers](https://github.com/koalaman/shellcheck/wiki/SC2199).

Reply:
I feel like the shellcheck authors might be willing to accept `[[ -n ${PATCHES[@]} ]]` or `[[ ${PATCHES[@]} ]]` as correct, since the "Problematic code" that they cite involves an incorrect comparison:

https://github.com/koalaman/shellcheck/wiki/SC2199#problematic-code

Reply:
Fixed in e42e9b61024d752a3dfb56ab8e8ea6936333ec58 to use the array length.